src/Stryker.Core/Stryker.Core.UnitTest/Mutants/TestResources/Mutator_SyntaxShouldBe_ConditionalStatement_IN.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace StrykerNet.UnitTest.Mutants.TestResources
+{
+    public class TestClass
+    {
+        void TestMethod()
+        {
+            int test = 10;
+            int testPlusTest = test + test;
+
+            bool Add(int x, int y) => x + y;
```

Comment:
Nitpick: this should return an int
```suggestion
            int Add(int x, int y) => x + y;
```